projectroles/tests/test_templatetags.py

```diff
@@ -618,3 +618,113 @@ def test_get_sidebar_app_legend(self):
         self.assertEqual(
             tags.get_sidebar_app_legend('Update Project'), 'Update<br />Project'
         )
+
+    def test_get_sidebar_links_home(self):
+        """Test get_sidebar_links() on the home view"""
+        url = reverse('home')
+        req_factory = RequestFactory()
```

Comment:
We can create this in `setUp()` as `self.req_factory` instead, so we'll save a few lines of code in re-initing it inside the test case methods.